magento/pwa-studio
# [feature] category breadcrumbs

## Description

* **Adds breadcrumbs to Category pages.**
    Breadcrumbs endpoint, after 2.3.4, will return `category_url_path` which we can use to generate the link for the breadcrumb. For now, this is just rendering Links and dividers.

## Related Issue





Closes [PWA-98](https://jira.corp.magento.com/browse/PWA-98).

## Acceptance 



### Verification Stakeholders
@schensley 
### Specification



### Verification Steps


1. From the landing page, click a category. You should see `Home`.
2. From left nav, click all the way into a category (like `Bottoms -> Skirts`). It should render and you should see breadcrumbs for it.
3. Do these things on desktop and mobile :)

## Screenshots / Screen Captures (if appropriate)
[![Image from Gyazo](https://i.gyazo.com/a986c4912e9765db6696c9fd8f1f6271.gif)](https://gyazo.com/a986c4912e9765db6696c9fd8f1f6271)
## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
